mu-plugins/global-fonts/index.php

```diff
@@ -108,36 +115,23 @@ function maybe_preload_font( $preload ) {
 /**
  * Return the details about a specific font face.
  */
-function get_font_urls( $font ) {
-	switch ( strtolower( $font ) ) {
-		case 'inter arrows':
-			return [ plugins_url( 'Inter/Inter-arrows.woff2', __FILE__ ) ];
-		case 'inter cyrillic':
-			return [ plugins_url( 'Inter/Inter-cyrillic.woff2', __FILE__ ), plugins_url( 'Inter/Inter-cyrillic-ext.woff2', __FILE__ ) ];
-		case 'inter greek':
-			return [ plugins_url( 'Inter/Inter-greek.woff2', __FILE__ ), plugins_url( 'Inter/Inter-greek-ext.woff2', __FILE__ ) ];
-		case 'inter latin':
-			return [ plugins_url( 'Inter/Inter-latin.woff2', __FILE__ ), plugins_url( 'Inter/Inter-latin-ext.woff2', __FILE__ ) ];
-		case 'inter vietnamese':
-			return [ plugins_url( 'Inter/Inter-vietnamese.woff2', __FILE__ ) ];
-		case 'eb garamond arrows':
-			return [ plugins_url( 'EB-Garamond/EBGaramond-arrows.woff2', __FILE__ ) ];
-		case 'eb garamond cyrillic':
-			return [ plugins_url( 'EB-Garamond/EBGaramond-cyrillic.woff2', __FILE__ ), plugins_url( 'EB-Garamond/EBGaramond-cyrillic-ext.woff2', __FILE__ ) ];
-		case 'eb garamond greek':
-			return [ plugins_url( 'EB-Garamond/EBGaramond-greek.woff2', __FILE__ ), plugins_url( 'EB-Garamond/EBGaramond-greek-ext.woff2', __FILE__ ) ];
-		case 'eb garamond latin':
-			return [ plugins_url( 'EB-Garamond/EBGaramond-latin.woff2', __FILE__ ), plugins_url( 'EB-Garamond/EBGaramond-latin-ext.woff2', __FILE__ ) ];
-		case 'eb garamond vietnamese':
-			return [ plugins_url( 'EB-Garamond/EBGaramond-vietnamese.woff2', __FILE__ ) ];
-		case 'eb garamond cyrillic italic':
-			return [ plugins_url( 'EB-Garamond/EBGaramond-Italic-cyrillic.woff2', __FILE__ ), plugins_url( 'EB-Garamond/EBGaramond-Italic-cyrillic-ext.woff2', __FILE__ ) ];
-		case 'eb garamond greek italic':
-			return [ plugins_url( 'EB-Garamond/EBGaramond-Italic-greek.woff2', __FILE__ ), plugins_url( 'EB-Garamond/EBGaramond-Italic-greek-ext.woff2', __FILE__ ) ];
-		case 'eb garamond latin italic':
-			return [ plugins_url( 'EB-Garamond/EBGaramond-Italic-latin.woff2', __FILE__ ), plugins_url( 'EB-Garamond/EBGaramond-Italic-latin-ext.woff2', __FILE__ ) ];
-		case 'eb garamond vietnamese italic':
-			return [ plugins_url( 'EB-Garamond/EBGaramond-Italic-vietnamese.woff2', __FILE__ ) ];
+function get_font_url( $font, $subset ) {
+	$lower_font = strtolower( trim( $font ) );
+	$lower_subset = strtolower( trim( $subset ) );
```

Comment:
What happens if one of these is not a valid value?

Reply:
if `$font` is not a valid value, `get_font_url` would return `false` with the `switch case` below.
if `$subset` is invalid, for example, an int `123`, 404 would be returned.
![image](https://user-images.githubusercontent.com/18050944/199331169-a80b0e70-17a6-4336-861e-6b8c381b9384.png)